website/content/docs/operations/federation/index.mdx

```diff
@@ -0,0 +1,53 @@
+---
+layout: docs
+page_title: Federated Cluster Operations
+description: Information on running Nomad federated clusters.
+---
+
+## Operational Considerations
+
+When operating multi-region federated Nomad clusters, the following considerations are important to
+keep in mind:
+
+* **Regular snapshots**: Nomad server state can be backed up using the
+  [`nomad operator snapshot save`][] and [`nomad operator snapshot agent`][] commands. Performing
+  regular backups can expedite disaster recovery and the cadence depends on cluster rates of change
+  and your internal SLA’s. Snapshots should also be tested regularly using the
+  [`nomad operator snapshot restore`][] command to ensure they work.
+
+* **Local ACL management tokens**: In order to perform federated cluster administration when the
+  authoritative region is down, local management tokens are required. It is important to have
+  existing “break glass” tokens for each region available.
+
+* **Known paths to creating local ACL tokens**: If the authoritative region fails, creation of
+  global ACL tokens will fail. If this happens, having the ability to create local ACL tokens will
+  allow you to continue to interact with each available federated region.
+
+## FAQ
+
+* **Can the authoritative region be deployed with servers only?** Yes, deploying the Nomad
+  authoritative region with servers only (no Nomad clients) will work as expected. It can expedite
+  disaster recovery of the region because restoration does not include objects such as nodes, jobs,
+  or allocations which are large and require compute intensive reconciliation after restoration.
+
+* **Can I migrate the authoritative region to a currently federated region?** It is possible by
+  following the steps below:
+
+  * Update the [`authoritative_region`][] configuration parameter on the desired authoritative
+    region servers.
+
+  * Restart the server processes in the new authoritative region and ensure all data is present in
+    state as expected. If the network was partitioned as part of the failure of the original
+    authoritative region, writes of replicated objects may not have been successfully replicated to
+    federated regions.
+
+  * Update the [`authoritative_region`][] configuration parameter on the federated region servers
+    and restart their processes.
```

Comment:
```suggestion
  1. Update the [`authoritative_region`][] configuration parameter on the
    desired authoritative region servers.
  1. Restart the server processes in the new authoritative region and ensure all
    data is present in state as expected. If the network was partitioned as part
    of the failure of the original authoritative region, writes of replicated
    objects may not have been successfully replicated to federated regions.
  1. Update the [`authoritative_region`][] configuration parameter on the
    federated region servers and restart their processes.
```

changed to an ordered list since I assumed the steps should be performed in order